### ISSUE TITLE
XmlPrimary.java:58-60: Create a unit test to verify that...

### ISSUE THREAD

**Issue by 0pdd** [artipie/rpm-adapter#83]
The puzzle `69-e192dc9a` from #69 has to be resolved: 

https://github.com/artipie/rpm-adapter/blob/493153a41dfc9dcae2bb43d8af290adf7d7acf47/src/main/java/com/artipie/rpm/meta/XmlPrimary.java#L58-L60

The puzzle was created by @g4s8 on 08-Apr-20. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
The job [#83](https://github.com/artipie/rpm-adapter/issues/83) assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/839bb023-4173-41cd-b3aa-72675d5bdaa4); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by victornoel** [comment]
@0crat wait for #86 to be merged

**Comment by 0pdd** [comment]
The puzzle `69-e192dc9a` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#117](https://github.com/artipie/rpm-adapter/issues/117) is still not solved.

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@2053453735b3, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-impediments.xsl says: The impediment for order/job "gh:artipie/rpm-adapter#83" exists in impediments.xml, but the order is absent among 41 orders.
```

[0.53.2](https://github.com/zerocracy/farm): CID: [aad77234-4f50-46cc-87af-f6f51f683ab8](https://www.0crat.com/footprint/CT2E6TK9B/aad77234-4f50-46cc-87af-f6f51f683ab8), Type: "Finish order"

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
@victornoel[/z](https://www.0crat.com/u/victornoel) this job was assigned to you 7days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@ce4a654318ed, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.7](https://github.com/zerocracy/farm): CID: [82785887-942c-4e09-a79b-7d73099ddda7](https://www.0crat.com/footprint/CT2E6TK9B/82785887-942c-4e09-a79b-7d73099ddda7), Type: "Cancel order"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@7d3334767964, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [19a21ea6-0d33-49bd-a08f-f8148a8f8313](https://www.0crat.com/footprint/CT2E6TK9B/19a21ea6-0d33-49bd-a08f-f8148a8f8313), Type: "Cancel order"

**Comment by victornoel** [comment]
@g4s8 see here for example

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@7d3334767964, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [7c2d70b4-421e-417c-8f3c-17a357615a45](https://www.0crat.com/footprint/CT2E6TK9B/7c2d70b4-421e-417c-8f3c-17a357615a45), Type: "Cancel order"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@7d3334767964, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [1c70ec68-35b9-4576-a198-ccc334c17598](https://www.0crat.com/footprint/CT2E6TK9B/1c70ec68-35b9-4576-a198-ccc334c17598), Type: "Cancel order"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@7d3334767964, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [5755bc1f-2374-4023-b8a9-1984eae7042c](https://www.0crat.com/footprint/CT2E6TK9B/5755bc1f-2374-4023-b8a9-1984eae7042c), Type: "Cancel order"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@7d3334767964, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [5db0c3eb-b284-45bf-8e57-d561685c993b](https://www.0crat.com/footprint/CT2E6TK9B/5db0c3eb-b284-45bf-8e57-d561685c993b), Type: "Cancel order"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@7d3334767964, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [86fa8a39-a10b-4eab-a7da-adb2fa9c1d3c](https://www.0crat.com/footprint/CT2E6TK9B/86fa8a39-a10b-4eab-a7da-adb2fa9c1d3c), Type: "Cancel order"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@ce4a654318ed, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.7](https://github.com/zerocracy/farm): CID: [ef4d4f04-876a-4df6-8ac6-e770121acc7c](https://www.0crat.com/footprint/CT2E6TK9B/ef4d4f04-876a-4df6-8ac6-e770121acc7c), Type: "Cancel order"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@7d3334767964, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [34f07b6f-d7e3-4d9c-bea9-f3e26665d18e](https://www.0crat.com/footprint/CT2E6TK9B/34f07b6f-d7e3-4d9c-bea9-f3e26665d18e), Type: "Cancel order"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@7d3334767964, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [4a2e3998-a075-4f0c-811a-4bf5b57d9b84](https://www.0crat.com/footprint/CT2E6TK9B/4a2e3998-a075-4f0c-811a-4bf5b57d9b84), Type: "Cancel order"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@7d3334767964, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [be046518-9aea-4e5b-8beb-97b33b7b74ef](https://www.0crat.com/footprint/CT2E6TK9B/be046518-9aea-4e5b-8beb-97b33b7b74ef), Type: "Cancel order"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@ce4a654318ed, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.7](https://github.com/zerocracy/farm): CID: [88952a22-ea3f-4663-b1ee-42e4ad6c20bb](https://www.0crat.com/footprint/CT2E6TK9B/88952a22-ea3f-4663-b1ee-42e4ad6c20bb), Type: "Cancel order"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@ce4a654318ed, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.7](https://github.com/zerocracy/farm): CID: [4f9ab9c5-27a4-4f3d-bac8-0e6abd65a28e](https://www.0crat.com/footprint/CT2E6TK9B/4f9ab9c5-27a4-4f3d-bac8-0e6abd65a28e), Type: "Cancel order"

**Comment by victornoel** [comment]
@0crat wait

**Comment by victornoel** [comment]
@0crat out

**Comment by victornoel** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat wait ([here](https://github.com/artipie/rpm-adapter/issues/83#issuecomment-616980725))

@victornoel There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@8362b1b91be7, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [c98c1bfc-4985-4008-99f4-2ae61738ffd1](https://www.0crat.com/footprint/CT2E6TK9B/c98c1bfc-4985-4008-99f4-2ae61738ffd1), Type: "Register impediment"

**Comment by 0crat** [comment]
@victornoel[/z](https://www.0crat.com/u/victornoel) the issue is closed not by @0pdd[/z](https://www.0crat.com/u/0pdd) (its creator); I won't close the order; please, re-open it and ask @0pdd[/z](https://www.0crat.com/u/0pdd) to close it

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/rpm-adapter/issues/83#issuecomment-616980825))

@victornoel There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@8362b1b91be7, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-estimates.xsl says: The estimate for job "gh:artipie/maven-adapter#74" exists in estimates.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [e43366ea-45c2-4f81-a3cd-6fd2e59adc3f](https://www.0crat.com/footprint/CT2E6TK9B/e43366ea-45c2-4f81-a3cd-6fd2e59adc3f), Type: "Cancel order"

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/artipie/rpm-adapter/issues/83#issuecomment-616980768))

@victornoel Job [#83](https://github.com/artipie/rpm-adapter/issues/83) is not in the agenda of @victornoel[/z](https://www.0crat.com/u/victornoel), can't set title

**Comment by victornoel** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/artipie/rpm-adapter/issues/83#issuecomment-616981985))

@victornoel There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@8362b1b91be7, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-orders.xsl says: The order for job "gh:artipie/maven-adapter#74" exists in orders.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [3281d8a6-99c8-445e-bb36-05dca91de955](https://www.0crat.com/footprint/CT2E6TK9B/3281d8a6-99c8-445e-bb36-05dca91de955), Type: "Check job status"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@8362b1b91be7, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-orders.xsl says: The order for job "gh:artipie/maven-adapter#74" exists in orders.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [f8829f6e-7cb1-417b-9d28-9352f73f5bca](https://www.0crat.com/footprint/CT2E6TK9B/f8829f6e-7cb1-417b-9d28-9352f73f5bca), Type: "Cancel order"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@8362b1b91be7, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-orders.xsl says: The order for job "gh:artipie/maven-adapter#74" exists in orders.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [7161a1d5-e66a-46a2-ac58-82d6322a3e01](https://www.0crat.com/footprint/CT2E6TK9B/7161a1d5-e66a-46a2-ac58-82d6322a3e01), Type: "Cancel order"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@8362b1b91be7, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-orders.xsl says: The order for job "gh:artipie/maven-adapter#74" exists in orders.xml, but the job is absent in the WBS among 52 jobs.
```

[0.53.9](https://github.com/zerocracy/farm): CID: [c39db8e8-1498-4058-8298-4012f9d02dbb](https://www.0crat.com/footprint/CT2E6TK9B/c39db8e8-1498-4058-8298-4012f9d02dbb), Type: "Cancel order"

**Comment by 0crat** [comment]
The user @victornoel[/z](https://www.0crat.com/u/victornoel) resigned from [#83](https://github.com/artipie/rpm-adapter/issues/83), please stop working. Reason for job resignation: It is older than 10 days, see [§8](http://www.zerocracy.com/policy.html#8)

**Comment by 0crat** [comment]
Job `gh:artipie/rpm-adapter#83` is not assigned, can't get performer

**Comment by g4s8** [comment]
@0crat pay 35min @victornoel

**Comment by 0crat** [comment]
The job [#83](https://github.com/artipie/rpm-adapter/issues/83) assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/4630f304-6526-4733-9c38-65c1b1655c4c); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@0eb3476e46c5, thread: pool-7-thread-1
com.jcabi.http.wire.RetryWire[95] java.io.IOException: failed after 4 attempts
```

[0.53.10](https://github.com/zerocracy/farm): Issue: artipie/rpm-adapter#83, Action: `created`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@0eb3476e46c5, thread: pool-9-thread-1
com.jcabi.http.wire.RetryWire[95] java.io.IOException: failed after 4 attempts
```

[0.53.10](https://github.com/zerocracy/farm): Issue: artipie/rpm-adapter#83, Action: `created`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@0eb3476e46c5, thread: pool-10-thread-1
com.jcabi.http.wire.RetryWire[95] java.io.IOException: failed after 4 attempts
```

[0.53.10](https://github.com/zerocracy/farm): Issue: artipie/rpm-adapter#83, Action: `created`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@0eb3476e46c5, thread: pool-11-thread-1
com.jcabi.http.wire.RetryWire[95] java.io.IOException: failed after 4 attempts
```

[0.53.10](https://github.com/zerocracy/farm): Issue: artipie/rpm-adapter#83, Action: `created`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@0eb3476e46c5, thread: pool-12-thread-1
com.jcabi.http.wire.RetryWire[95] java.io.IOException: failed after 4 attempts
```

[0.53.10](https://github.com/zerocracy/farm): Issue: artipie/rpm-adapter#83, Action: `created`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@0eb3476e46c5, thread: pool-13-thread-1
com.jcabi.http.wire.RetryWire[95] java.io.IOException: failed after 4 attempts
```

[0.53.10](https://github.com/zerocracy/farm): Issue: artipie/rpm-adapter#83, Action: `created`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@0eb3476e46c5, thread: pool-14-thread-1
com.jcabi.http.wire.RetryWire[95] java.io.IOException: failed after 4 attempts
```

[0.53.10](https://github.com/zerocracy/farm): Issue: artipie/rpm-adapter#83, Action: `created`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@0eb3476e46c5, thread: pool-15-thread-1
com.jcabi.http.wire.RetryWire[95] java.io.IOException: failed after 4 attempts
```

[0.53.10](https://github.com/zerocracy/farm): Issue: artipie/rpm-adapter#83, Action: `created`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@0eb3476e46c5, thread: pool-16-thread-1
com.jcabi.http.wire.RetryWire[95] java.io.IOException: failed after 4 attempts
```

[0.53.10](https://github.com/zerocracy/farm): Issue: artipie/rpm-adapter#83, Action: `created`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@0eb3476e46c5, thread: pool-17-thread-1
com.jcabi.http.wire.RetryWire[95] java.io.IOException: failed after 4 attempts
```

[0.53.10](https://github.com/zerocracy/farm): Issue: artipie/rpm-adapter#83, Action: `created`

**Comment by carlosmiranda** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/rpm-adapter/issues/83#issuecomment-618153584))

@carlosmiranda Job `gh:artipie/rpm-adapter#83` is not assigned, can't get performer

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/artipie/rpm-adapter/issues/83#issuecomment-618288446))

@g4s8 This is what I know about this job in CT2E6TK9B, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#83](https://github.com/artipie/rpm-adapter/issues/83) is not in scope
  * The job is waiting quality review verdict by @sereshqua[/z](https://www.0crat.com/u/sereshqua) for 1days
  * @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) will receive 35 minutes after quality review
  * Job [footprint](https://www.0crat.com/footprint/CT2E6TK9B?q=%7Bjob%3A%27gh%3Aartipie%2Frpm-adapter%2383%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by g4s8** [comment]
@0crat refuse

**Comment by g4s8** [comment]
@0crat out

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/artipie/rpm-adapter/issues/83#issuecomment-618294047))

@g4s8 Job `gh:artipie/rpm-adapter#83` is not assigned, can't get performer

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/rpm-adapter/issues/83#issuecomment-618293996))

@g4s8 Job `gh:artipie/rpm-adapter#83` is not assigned, can't get performer

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#219](https://github.com/artipie/rpm-adapter/issues/219), [#220](https://github.com/artipie/rpm-adapter/issues/220) are still not solved; solved: [#117](https://github.com/artipie/rpm-adapter/issues/117).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#254](https://github.com/artipie/rpm-adapter/issues/254), [#255](https://github.com/artipie/rpm-adapter/issues/255) are still not solved; solved: [#117](https://github.com/artipie/rpm-adapter/issues/117), [#219](https://github.com/artipie/rpm-adapter/issues/219), [#220](https://github.com/artipie/rpm-adapter/issues/220).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#255](https://github.com/artipie/rpm-adapter/issues/255), [#366](https://github.com/artipie/rpm-adapter/issues/366) are still not solved; solved: [#117](https://github.com/artipie/rpm-adapter/issues/117), [#219](https://github.com/artipie/rpm-adapter/issues/219), [#220](https://github.com/artipie/rpm-adapter/issues/220), [#254](https://github.com/artipie/rpm-adapter/issues/254).

**Comment by 0pdd** [comment]
@0pdd all 6 puzzles are solved here: [#117](https://github.com/artipie/rpm-adapter/issues/117), [#219](https://github.com/artipie/rpm-adapter/issues/219), [#220](https://github.com/artipie/rpm-adapter/issues/220), [#254](https://github.com/artipie/rpm-adapter/issues/254), [#255](https://github.com/artipie/rpm-adapter/issues/255), [#366](https://github.com/artipie/rpm-adapter/issues/366).
